README.md

```diff
@@ -587,6 +587,7 @@ Below is the complete list of parameters that can be set using environment varia
 - **REDMINE_RELATIVE_URL_ROOT**: The relative url of the Redmine server, e.g. `/redmine`. No default.
 - **REDMINE_ATTACHMENTS_DIR**: The attachments directory. Defaults to `/home/redmine/data/files`
 - **REDMINE_SECRET_TOKEN**: Secret key for verifying cookie session data integrity. Defaults to a random alphanumeric string.
+- **REDMINE_RMAGICK_FONT_PATH**: The rmagick_font_path for the png export function of GANTT to work. No default.
```

Comment:
This person says they set their path to /usr/share/fonts/truetype/dejavu/DejaVuSans.ttf which does exist in our container. Would this be an acceptable default?
https://github.com/docker-library/redmine/issues/65#issuecomment-297280735

I'd prefer we set it to something reasonable instead of requiring everyone to set it themselves

Reply:
@jcormier 
I think the reason why the default value isn't set in Redmine is to respond flexibly to the environment.
When using containers, we do not need to do it.
So, I think setting the default value might be reasonable.

In the container, the default font is `DejaVuSans.ttf: "DejaVu Sans" "Book"`.
Do you mind if I set it as default?

```
# fc-match
DejaVuSans.ttf: "DejaVu Sans" "Book"
# fc-list | grep DejaVuSans.ttf
/usr/share/fonts/truetype/dejavu/DejaVuSans.ttf: DejaVu Sans:style=Book
```

Reply:
Sounds good, please set as default.  Thanks

Reply:
I have one more question.
Do you think I should add `REDMINE_RMAGICK_FONT_PATH` to `docker-compose.yml`?

Reply:
probably not, I think the default will be good for almost everyone.  They can add it if they want to change it.

Reply:
@jcormier 
I've pushed
Could you please review it?